org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/ProjectStorageUpdateJob.java

```diff
@@ -42,7 +42,7 @@ protected IStatus run(IProgressMonitor monitor) {
     try {
       var server = SonarLintCorePlugin.getServersManager().findById(connectionId);
       server.ifPresent(s -> {
-        s.updateProjectStorage(projectKey, null, monitor);
+        s.updateProjectStorage(projectKey, monitor);
         s.sync(Set.of(projectKey), monitor);
```

Comment:
Shouldn't we also sync issues & taints here ?

Reply:
The `sync` method is taking care of synchronizing issues as well